Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs

```diff
@@ -112,15 +115,22 @@ view push state =
       , recentRideView state push
       , headingView state (getString ALL_TOPICS)
       , allTopicsView state push
-      , apiFailureView state push
+      , apiFailureView state push 
       ]
     , linearLayout
       [ width MATCH_PARENT
       , height MATCH_PARENT
       , visibility if state.props.isCallConfirmation then VISIBLE else GONE
       ][PopUpModal.view (push <<< PopupModelActionController) (callConfirmationPopup state)]
-
-  ]
+    , deleteAccountView state push
+  ] <> if checkAccountStatus state ST.CONFIRM_REQ || checkAccountStatus state ST.DEL_REQUESTED then 
+    [ linearLayout
+      [ width MATCH_PARENT
+      , height MATCH_PARENT
+      , background Color.lightBlack900
+      ]( [] <> if checkAccountStatus state ST.CONFIRM_REQ then [PopUpModal.view (push <<<  PopUpModalAction) (requestDeletePopUp state )]
+          else if checkAccountStatus state ST.DEL_REQUESTED then [PopUpModal.view (push <<< AccountDeletedModalAction) (accountDeletedPopUp state)]
```

Comment:
no need to use `checkAccountStatus ` as function and import, instead you can do

```
let checkAccountStatus = ...
in
    linearLayout[][]
```

and use it inside your view